src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -89,7 +92,6 @@ export const styles = StyleSheet.create({
     fontSize: normalizeFontSize(24),
     lineHeight: normalizeFontSize(29.05),
     color: '#000',
-    width: '100%',
     paddingHorizontal: dimen.normalizeDimen(20),
     fontWeight: 'bold'
   },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `width: '100%'` to `color: '#000'` seems to be a mistake. The `width` property sets the width of the element while `color` sets the text color. If you want to set both properties, they should be on separate lines.
```diff
-    width: '100%',
+    color: '#000',
     paddingHorizontal: dimen.normalizeDimen(20),
     fontWeight: 'bold'
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->